### PR TITLE
enhance createimage utility

### PR DESCRIPTION
**Pull request by jichenjc**
Signed-off-by: jichenjc <jichenjc@cn.ibm.com>

### CODE REVIEW

File: zthin-parts/zthin/lib/zthinshellutils [hunk 1]
```diff
@@ -710,8 +710,9 @@ function connectDisk {
     if [[ $attempts -gt 120 ]]; then
       return 5
     fi
-    if [[ $(( $attempts % 30 )) -eq 0 ]]; then
+    if [[ $(( $attempts % 5 )) -eq 0 ]]; then
```
Comment (bjhuangr) [inline zthin-parts/zthin/lib/zthinshellutils:713]: increase the frequency of printing attempt connect?

File: zthin-parts/zthin/lib/zthinshellutils [hunk 2]
```diff
@@ -710,8 +710,9 @@ function connectDisk {
...
       warn "Waiting on lock: ${diskConnectionLock}" | sed 's/^/  /'
+      warn "Manual remove ${diskConnectionLock} if you are sure no concurrent actions to unblock"
```
Comment (bjhuangr) [inline zthin-parts/zthin/lib/zthinshellutils:715]: does this statement always reasonable?

Comment (jichenjc) [reply zthin-parts/zthin/lib/zthinshellutils:715]: yes, I said `you are sure` no concurrent actions

so mostly we should not enter here but sometimes due to previous error the connect action stuck here forever ..

Comment (dongyanyang) [reply zthin-parts/zthin/lib/zthinshellutils:715]: If the while loop can exit normally, actually we can just print this before line 711.
(I tried that this loop can work normally, can exit after 120 attempts)

Comment (jichenjc) [reply zthin-parts/zthin/lib/zthinshellutils:715]: ok , then let me move this

**Pull request closed without merging**